Kickstarter-iOS/Views/Cells/LiveStreamDiscoveryCell.swift

```diff
@@ -0,0 +1,134 @@
+import AlamofireImage
+import Library
+import LiveStream
+import Prelude
+
+internal final class LiveStreamDiscoveryCell: UITableViewCell, ValueCell {
+  private let viewModel: LiveStreamDiscoveryCellViewModelType = LiveStreamDiscoveryCellViewModel()
+
+  @IBOutlet private weak var backgroundImageView: UIImageView!
+  @IBOutlet private weak var blurView: UIView!
+  @IBOutlet private weak var countdownStackView: UIStackView!
+  @IBOutlet private var colonViews: [UILabel]!
```

Comment:
Oh, that's a collection of outlets, we just apply the same style to a bunch of labels (the colons in the countdown).